force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html

```diff
@@ -0,0 +1,64 @@
+<template>
+    <template if:true={loaded}>
+        <div class="c-container" >
+            <lightning:layout >
+                <div class= "slds-text-heading_small slds-truncate" >
+                    <lightning-card title={title}>
+                        <div class="slds-modal__content slds-text-body_regular slds-hyphenate">
+                            <div class="slds-scrollable_y"  style={boxStyle}>
+                                <table class="slds-table " aria-describedby="To display User's Active Credentils">
+                                    <thead>
+                                    </thead> 
+                                    <tbody>
+                                        <template for:each={userCredentialsData} for:item="u">  
+                                            <tr class="slds-hint-parent" key = {u.Id} >
+                                                <th></th>
+                                                <td data-label="Credentials">                                                    
+                                                    <lightning-layout vertical-align="center" horizontal-align="left" >
+                                                        <lightning-layout-item flexibility="auto" size="3" padding="horizontal-Small">
+                                                            <div class="slds-box slds-align_absolute-center" style="border:0px;float:left;margin-left:-33px;padding-bottom:0px">
```

Comment:
move all the inline styling to css class

Reply:
Changes updated  please review